pkgs/development/python-modules/natural/default.nix

```
@@ -0,0 +1,29 @@
+{ buildPythonPackage
+, fetchFromGitHub
+, lib
+, six
+, django
+}:
+buildPythonPackage rec {
+  pname = "natural";
+  version = "0.2.0";
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    owner = "tehmaze";
+    repo = "natural";
+    rev = "refs/tags/${version}";
+    hash = "sha256-DERFKDGVUPcjYAxiTYWgWkPp+Myd/9CNytQWgRya570=";
+  };
+
+  buildInputs = [ django ];
```

```suggestion
  nativeCheckInputs = [ django ];
```
https://github.com/tehmaze/natural/blob/4d41ed6708965420ad884339e0e9ca5f089b2202/requirements_development.txt#L1
`Django` is not marked as a runtime dependency.